handlers/boilerplatecode.ts

```diff
@@ -1,11 +1,11 @@
 import { Handler, Context } from "aws-lambda";
-import { download } from "../services/compileGRPCStubs";
+import { generateNodejsBoilerplatecode } from "../services/compileGRPCStubs";
 
 export const create: Handler = async (
   event: { serviceId: string; orgId: string },
   context: Context
-): Promise<Context> => {
+): Promise<any> => {
```

Comment:
We should not use `any`.
I guess we can get the particular type from aws-lambda package itself. 

Else write a custom interface and add it in the return type.

Reply:
Updated in the latest commit